test/cdk/stacks/core-api-base.test.ts

```diff
@@ -112,6 +112,6 @@ describe.each(regions)('API Core Nag Pack Tests | Region Test: %s', (awsRegion)
 
     test('NIST800.53r5 CDK NAG Errors', () => {
         const errors = Annotations.fromStack(stack).findError('*', Match.stringLikeRegexp('NIST.*'));
-        expect(errors.length).toBe(6);
+        expect(errors.length).toBe(7);
```

Comment:
This change is a complaint that the authorizer is using an inline policy as a result of adding in `managementKeySecret.grantRead(authorizerLambda);`